src/features/comments/components/Comment.tsx

```diff
@@ -0,0 +1,178 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  Typography,
+  IconButton,
+  Button,
+  Avatar,
+} from "@mui/material";
+import { AddComment, Favorite } from "@mui/icons-material";
+import { BaseCommentPropType } from "../models/Comments.model";
+import useTheme from "features/theme/useTheme";
+import { getSecureUrl } from "shared/utils/cloudinaryUtil";
+
+const BaseComment = ({ data }: any) => {
```

Comment:
Can't we use BaseCommentModel instead of any?

Reply:
I wasn’t sure if there would be conflicting types but you right